ds-intermediate-python/syllabus.md

```diff
@@ -0,0 +1,29 @@
+Setting up Environment For Collaborative Code Development
+* Introduction to Our Software Project
+* Virtual Environments For Software Development
+* Integrated Software Development Environments
+* Collaborative Software Development Using Git and gitHub
+* Python Code Style Conventions
+* Verifying Code Style Using Linters
+
+Ensuring Correctness of Software at Scale
+* Automatically Testing Software
+* Scaling Up Unit Testing
+* Continuous Integration for Automated Testing
+* Diagnosing Issues and Improving Robustness
+
+Software Development as a Process
+* Software Requirements
+* Software Architecture and Design
+* Programming Paradigms
+* Functional Programming Intuition
+* Object Oriented Programming Intuition
```

Comment:
Here I would like to shorten. Instead of following the material, we do:
* [functional programming](https://carpentries-incubator.github.io/python-intermediate-development/34-functional-programming/index.html) only an intuition. Perhaps only the first exercise.
* [https://carpentries-incubator.github.io/python-intermediate-development/35-object-oriented-programming/index.html] only an intuition. How does OOP work and why would you use it, without going into the details.
* Skip https://carpentries-incubator.github.io/python-intermediate-development/36-architecture-revisited/index.html

Reply:
Cool. Yes there is still a lot about software design, including modularity, reusability, etc.